src/content/structured/accessibility/requirement/index.mdx

```diff
@@ -16,7 +16,7 @@ contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/struc
 
 ## Introduction to the Policy
 
-All digital products that the UK Intelligence Community builds, buys or uses must meet the Intelligence Community's Digital Accessibility Policy. This includes mobile, browser and desktop based software, interfaces attached to physical hardware and operating systems.
+All digital products that the UK Intelligence Community builds, buys or uses must meet the Intelligence Community's Digital Accessibility Policy. This includes mobile-, browser- and desktop-based software, interfaces attached to physical hardware and operating systems.
```

Comment:
i'm not sure about the dashes on mobile and desktop here - is there precedent for fragments of a hyphenated phrase keeping their hyphens?